contracts/token/ERC721/ERC721.sol

```diff
@@ -81,9 +81,10 @@ contract ERC721 is ERC165, IERC721 {
      * @param tokenId uint256 ID of the token to be approved
      */
     function approve(address to, uint256 tokenId) public {
+        require(_exists(tokenId), "tokenId to approve doesn't exist");
```

Comment:
What is the reason for this extra `require`? `ownerOf` already asserts this.

Reply:
Yes right. I will remove it.